frontend/src/modules/Dashboard/Components/Dashboard.tsx

```diff
@@ -74,9 +84,7 @@ export const Dashboard = () => {
                 membership.groupNumber !== -1
                   ? `${course.names.join('/')}_${membership.groupNumber}`
                   : undefined,
-              dateShareMatchEmail: group?.shareMatchEmailTimestamp?.toLocaleString(),
-              dateCheckInEmail: group?.checkInEmailTimestamp?.toLocaleString(),
-              dateAddStudentEmail: group?.addStudentEmailTimestamp?.toLocaleString(),
+              dateTemplateEmails: localeMap(group?.templateTimestamps),
```

Comment:
Don't think this is what they want

<img width="438" alt="Screen Shot 2022-08-13 at 02 06 53" src="https://user-images.githubusercontent.com/22627336/184477080-964bbebb-cee5-4cec-b7bf-e898285c7a3a.png">